tools/json_schemas_to_typescript.rb

```diff
@@ -96,8 +96,9 @@ def write_files(changed: nil, added: nil, removed: nil)
 
               main_interface =
                 file_content.
-                  match(/^export interface (?<main_interface>\w+) {/).
-                  []('main_interface')
```

Comment:
I think that there is a bit of a bug in `Layout/SpaceBeforeBrackets`, which was complaining about this code, but I was able to work around that bug via this change, which I think is probably an improvement, anyway.